### PR TITLE
Gym Landing Page

### PR DESCRIPTION
**Pull request by theanimator20**
# Zero To Mastery - Webblocks 2022 📦📦

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
i have added a landing page for a local gym so people can join the gym entering their details online easily.

icon

### CODE REVIEW

**Comment by theanimator20** [comment]
Hello Please merge my PR with hacktoberfest label

File: src/blocks/Keegan/css/style.css [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+
+
+
+
+
+
+
+
+/* {
```
Comment (sophiabrandt) [inline src/blocks/Keegan/css/style.css:143]: Please remove outdated code.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Please provide licensing details to the images you used.

Images are normally under copyright, so we cannot merge your PR until we know that you are legally allowed to use those images.

File: src/blocks/Keegan/css/style.css (hunk 1 above)
Comment (theanimator20) [reply src/blocks/Keegan/css/style.css:143]: @sophiabrandt  i have downloaded from google which are available to anyone u can add it.

**Review by theanimator20 — COMMENTED** [review]
Removed the outdated code

**Comment by sophiabrandt** [comment]
> Removed the outdated code

Thank you. What about the image licences?

**Comment by theanimator20** [comment]
Can u tell how to get an image licence ??.. i downloaded them from google (open source licence right?)

**Comment by auxfuse** [comment]
Downloading images from Google does not give you attribution or license rights. Please replace your images with copyright free images @theanimator20 - for reference, you could get similar images that are copyright-free from Pexels/Unsplash.

**Comment by theanimator20** [comment]
Thank you i will replace those images

**Comment by theanimator20** [comment]
I have changed the images used images from unsplash
icon

File: src/blocks/Keegan/css/style.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
+* {
+    margin: 0px;
+    padding: 0px;
+}
+
+body {
+    height: 100vh;
+    font-family: 'Crete Round', serif;
+    /* border:5px red solid; */
```
Comment (sophiabrandt) [inline src/blocks/Keegan/css/style.css:9]: Can you please remove outdated code?

File: src/blocks/Keegan/css/style.css [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    /* background-color: #61890c; */
```
Comment (sophiabrandt) [inline src/blocks/Keegan/css/style.css:10]: Same here.

File: src/blocks/Keegan/css/style.css [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    background-image: url("/src/blocks/Keegan/img/img1.avif");
+    overflow: hidden;
+    /* background-size: 100vw; */
```
Comment (sophiabrandt) [inline src/blocks/Keegan/css/style.css:13]: Same here.

File: src/blocks/Keegan/css/style.css [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    top: 0px;
+    z-index: 10;
+}
+
+.navbar {
+
+    display: inline-block;
+    position: absolute;
+    right: 35%;
+    /* width: 33%; */
```
Comment (sophiabrandt) [inline src/blocks/Keegan/css/style.css:45]: Same here.

File: src/blocks/Keegan/css/style.css [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+    margin: 18px auto;
+    /* border:2px rebeccapurple solid; */
```
Comment (sophiabrandt) [inline src/blocks/Keegan/css/style.css:47]: Same here

File: src/blocks/Keegan/css/style.css [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+text-align: center;
+}
+
+.contact{
+    width:88%;
+}
+
+.gender{
+    margin:8px;
+    /* margin-right: 153px; */
```
Comment (sophiabrandt) [inline src/blocks/Keegan/css/style.css:127]: Same here.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

**Review by auxfuse — APPROVED** [review]
LGTM 🎉

**Pull request merged**